docker/windows/runner/Dockerfile

```diff
@@ -20,7 +21,7 @@ ARG RUNNER_VERSION
 RUN Invoke-WebRequest -Uri 'https://aka.ms/install-powershell.ps1' -OutFile install-powershell.ps1; \
     powershell -ExecutionPolicy Unrestricted -File ./install-powershell.ps1 -AddToPath
 
-RUN Invoke-WebRequest -Uri https://github.com/actions/runner/releases/download/v$env:RUNNER_VERSION/actions-runner-win-x64-$env:RUNNER_VERSION.zip -OutFile runner.zip; \
+RUN Invoke-WebRequest -Uri https://github.com/actions/runner/releases/download/v$env:RUNNER_VERSION/actions-runner-win-x64-$env:RUNNER_VERSION.zip -OutFile C:/runner.zip; \
```

Comment:
Tiny "style" nitpick for passersby for writing this kind of install and provisioning code in Dockerfiles: put the output files in some consistent temporary directory e.g. $TEMP or C:\tmp, and remove them later from there with a wildcard `rm -r $TEMP/*` or such. This usually ends up easier to maintain in the long run.

No need to change here as you didn't author this